test/unit_tests/models/api_calls_test.dart

```diff
@@ -0,0 +1,169 @@
+/// Copyright 2020 Google LLC
+///
+/// Licensed under the Apache License, Version 2.0 (the "License");
+/// you may not use this file except in compliance with the License.
+/// You may obtain a copy of the License at
+///
+///     https://www.apache.org/licenses/LICENSE-2.0
+///
+/// Unless required by applicable law or agreed to in writing, software
+/// distributed under the License is distributed on an "AS IS" BASIS,
+/// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+/// See the License for the specific language governing permissions and
+/// limitations under the License.
+
+import 'dart:io';
+import 'package:chrome_management_app/objects/account_devices.dart';
+import 'package:chrome_management_app/objects/detailed_device.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:mockito/mockito.dart';
+import 'package:http/http.dart' as http;
+import 'package:chrome_management_app/models/api_calls.dart';
+
+/// Class to mock a http client
+class MockClient extends Mock implements http.Client {}
+
+main() {
+  /// Mock a authentication token
+  final authToken = '123';
+
+  /// Mock a next page token
+  final nextPageToken = '456';
+
+  /// Mock a device ID
+  final deviceId = '789';
+
+  /// Url to get the intial page of the list of devices
```

Comment:
These 2 shouldn't be repeated, get them from the main class

Reply:
Done.